### PR TITLE
Tag BioSequences.jl v0.7.0

### PR DESCRIPTION
**Pull request by attobot**
Repository: [BioJulia/BioSequences.jl](https://github.com/BioJulia/BioSequences.jl)
Release: [v0.7.0](https://github.com/BioJulia/BioSequences.jl/releases/tag/v0.7.0)
Travis: [Travis Build Status](https://travis-ci.org/BioJulia/BioSequences.jl/builds/258353087)
Diff: [vs v0.6.3](https://github.com/BioJulia/BioSequences.jl/compare/f7f4d882f1c8475a06b264fc4a10bf8f14d16c7a...6e8a2f8697f73cb25e70bdc647d212dd4e84624f)
`requires` vs v0.6.3: 
```diff
--- 0.6.3/requires
+++ 0.7.0/requires
@@ -4,7 +4,6 @@
 BioSymbols
 BufferedStreams
 Combinatorics
-Compat 0.18
 IndexableBitVectors
 IterTools 0.1
 IntervalTrees
```
cc: @Ward9250

Please make sure that:
- CI passes for supported Julia versions (if applicable).
- Version bounds reflect minimum requirements.

### CODE REVIEW

**Comment by tkelman** [comment]
You still have an `import Compat: take!` in the tests. And looks like the reverse-dependencies are unhappy with the renaming of that type, should it use `@deprecate_binding` ?

File: BioSequences/versions/0.7.0/requires [hunk 1]
```diff
@@ -0,0 +1,11 @@
+julia 0.5
```
Comment (tkelman) [inline BioSequences/versions/0.7.0/requires]: looks like you forgot to change this to 0.6

**Comment by tkelman** [comment]
If Compat is removed from REQUIRE, it shouldn't be used here https://github.com/BioJulia/BioSequences.jl/blob/a5142983104c1f97d2d881884f3801600b7bef62/test/runtests.jl#L9

**Comment by tkelman** [comment]
Looks like at least GeneticVariation.jl would break from this. Can either upper bound that dependency, or add a `@deprecate_binding` for the renamed type so it's a warning instead of an error.

**Comment by Ward9250** [comment]
GeneticVariation.jl will also be released with a version which drops julia 0.5 support. To get around this I can work on GeneticVariation and tag a new release and then it and BioSequences can get released together?

**Comment by tkelman** [comment]
Or the metadata requires files of the existing GeneticVariation tags could be modified retroactively (just the requires files, leave sha files alone) to add a BioSequences 0.7 upper bound. Then anyone with GeneticVariation installed wouldn't upgrade BioSequences until there's a BioSequences-0.7-compatible version available. Either way.

**Pull request merged**